tests/queries/0_stateless/02864_statistics_bugs.sql

```diff
@@ -25,3 +25,13 @@ CREATE TABLE bug_67742 (a Int16 STATISTICS(tdigest)) Engine = MergeTree() ORDER
 INSERT INTO bug_67742 SELECT number FROM system.numbers LIMIT 10000;
 SELECT count(*) FROM bug_67742 WHERE a < '9999999999999999999999999';
 DROP TABLE bug_67742;
+
+SET allow_experimental_statistics = 1;
```

Comment:
I think it's better to create a new test with name that reflects verified functionality in a clearer way

Reply:
Okay, will do.